opendatalakehouse/00_prereqs.md

```diff
@@ -9,7 +9,7 @@ As part of the `Prerequisites`, we will walk through the initial setup and confi
 
 # Cloudera Machine Learning (CML) Project
 
-In this phase, we will be deploying an end\-to\-end machine learning project that will also be used in the [03_predict](03_predict.md) phases.
+In this phase, we will be deploying an end\-to\-end machine learning project that will also be used in the [03_predict](04_predict) phases.
```

Comment:
04_predict.md it should be here , it is not redirecting to the related readme file as .md is missing here